camayoc/tests/qcs/cli/utils.py

```diff
@@ -70,3 +70,24 @@ def cred_show(options, output, exitstatus=0):
     assert qpc_cred_show.expect(pexpect.EOF) == 0
     qpc_cred_show.close()
     assert qpc_cred_show.exitstatus == exitstatus
+
+
+def source_show(options, output, exitstatus=0):
+    """Show a source entry.
+
+    :param options: A dictionary mapping the option names and their values.
+        Pass ``None`` for flag options.
+    :param output: A regular expression that matches the expected output.
```

Comment:
Can you specify a bit more particularly what format you mean? I think you are saying it just has to be a string, but might be misread as a compiled `re` object

Reply:
This actually should be `a regular expression pattern string`. That is important because it will be compiled as a regular expression and any regex syntax issue will raise an exception that it is not able to compile.

This is related to what `output` should be conversation we have in one of the previous comments. Let's do the following, find the way output should be (it must be the same for all `*_show` helpers) and then I will update this properly. Later I will update the cred_show to match the output object like one for source_cred.